### PR TITLE
docs: Generate script from containerd install guide.

### PR DESCRIPTION
**Pull request by jcvenegas**
Update documentation to allow generate an script using
kata-doc-to-script.sh

Add ci script to quickly install `ctr` with kata
configured.

The generated script uses kata-deploy tarball,
so using make binary-tarball will allow test the
latest code.

example
```
$ ./ci/containerd-installer.sh
...
+ sudo ctr run --runtime io.containerd.kata.v2 --rm -t docker.io/library/busybox:latest test-kata sh -c 'echo "Hello from kata with kernel $(uname -r)"'
Hello from kata with kernel 5.10.25
```

### CODE REVIEW

File: ci/containerd-installer.sh
Comment (GabyCT) [inline ci/containerd-installer.sh:3]: 2021?

**Review by GabyCT — DISMISSED** [review]

File: ci/containerd-installer.sh
Comment (jcvenegas) [reply ci/containerd-installer.sh]: Ups, fixed , thanks @GabyCT

**Comment by jcvenegas** [comment]
@jodh-intel please take a look, I we dont have documentation test,  I wonder if it worth create one job that just run this script. Or keep updating our test to at some point those use this script and allow verify it is working all the time.

File: ci/containerd-installer.sh
Comment (jcvenegas) [inline ci/containerd-installer.sh:14]: this has an invalid URL

**Review by cmaf — DISMISSED** [review]

**Comment by fidencio** [comment]
Although this PR is good to go, I'm adding the do-not-merge label so @jodh-intel has a change to take a look at it.

File: docs/install/container-manager/containerd/containerd-install.md
Comment (jodh-intel) [inline docs/install/container-manager/containerd/containerd-install.md:95]: Nit: "Kata" (here and below).

File: docs/install/container-manager/containerd/containerd-install.md
Comment (jodh-intel) [inline docs/install/container-manager/containerd/containerd-install.md:120]: Hard-coded architecture. Could you use a `arch=$(uname -m)` variable?

File: docs/install/container-manager/containerd/containerd-install.md
Comment (jodh-intel) [inline docs/install/container-manager/containerd/containerd-install.md:122]: Might be clearer to add the implied dash before `xvf`:

```suggestion
        sudo tar -xvf ./containerd.tar.gz -C /
```

**Review by jodh-intel — COMMENTED** [review]
Thanks @jcvenegas. A few comments.

**Comment by jcvenegas** [comment]
@jodh-intel changes ready for new review

**Comment by jodh-intel** [comment]
Marking as stale as this PR may have been abandoned.

**Comment by jodh-intel** [comment]
Re-ping @jcvenegas !

**Comment by fidencio** [comment]
A head's up that may affect your PR.

In the Architecture Committee meeting from January 25th, 2022, the Architecture Committee has agreed on using the "Dismiss stale pull request approvals when new commits are pushed" configuration from GitHub.  It basically means that if your PR has been rebased or updated, the approvals given will be erased.

In order to minimize traumas due to the new approach, please, consider adding a note on the changes done before the rebase / force-push, and also pinging the reviewers for a subsequent round of reviews.

Thanks for your understanding!

Related issue: https://github.com/kata-containers/community/issues/249

**Comment by fidencio** [comment]
As this PR has been waiting for an update since October 2021, I'm closing it for now.

@jcvenegas, please, if you find some free cycles to work on this PR, please, just re-open it and let us know.
Thanks for the contribution and looking forward to seeing an up-to-date version of this PR.

**Pull request closed without merging**
